rpcudp/protocol.py

```diff
@@ -80,7 +81,16 @@ async def _accept_request(self, msg_id, data, address):
             return
 
         if not asyncio.iscoroutinefunction(func):
-            func = asyncio.coroutine(func)
+            # ref. https://stackoverflow.com/a/66490669
+            def awaitify(sync_func):
+                """Wrap a synchronous callable to allow ``await``'ing it"""
+                @wraps(sync_func)
+                async def async_func(*args, **kwargs):
+                    return sync_func(*args, **kwargs)
+                return async_func
+
+            func = awaitify(func)
```

Comment:
Is it possible that something like this would work?

```python
if not asyncio.iscoroutinefunction(func):
    response = func(address, *args)
else:
    response = await func(address, *args)
```